pyproject.toml

```diff
@@ -36,7 +36,9 @@ dependencies = [
     "scipy>=1.7.3",
     "tqdm>=4.62.3",
     "matplotlib>=3.3.4",
-    "typing_extensions; python_version <= '3.8'"
+    "typing_extensions; python_version <= '3.8'",
+    "transformers<=4.30.2; python_version == '3.7'",
```

Comment:
I wonder if this should be a part of the base dependencies. I think it is better fitted to add it under 'torch' (see line 78/80 and below)?